notes/BUFRCREX_CodeFlag_table.csv

```diff
@@ -216,6 +216,15 @@ tableNo,tableName,noteID,notation
 0 08 091,Coordinates significance,,
 0 08 092,Measurement uncertainty expression,,
 0 08 093,Measurement uncertainty significance,,
+0 08 093,Method used to calculate the average daily temperature,,
```

Comment:
This is a typo - the FXY number for "Method used to calculate the average daily temperature" is 0-08-094, not 0-08-093.

Reply:
Otherwise, everything looks OK to me.  Please add another commit to fix the typo, and then I'll be happy to give review approval.

Reply:
thanks for noticing